tests/main/interfaces-system-packages-doc/test-snapd-app/meta/snap.yaml

```diff
@@ -0,0 +1,7 @@
+name: test-snapd-app
```

Comment:
For debugging, it seems this could maybe have a more descriptive name. Not a blocker, just an observation.

Reply:
I tend to use `test-snapd-app` for snaps specific to a given test because it's just shorter than a name involving the test name. Some tests use that and it was tedious so I moved away from that idea.